### ISSUE TITLE
Terrible performance with Intel graphics on macOS 11

### ISSUE THREAD

**Issue by StarWitch** [taisei-project/taisei#309]
It seems like there's a major regression with Intel graphics on macOS 11, even on v1.3.2. It runs extremely slowly, even with post-processing disabled.

**Comment by Akaricchi** [comment]
I don't think we can do much about this. It's a case of the hardware being too weak to render at the desired resolution, which isn't unique to intel iGPUs or 4k screens. We have performance settings for exactly that reason.

It would be nice to implement some kind of dynamic resolution scaling system as modern games often do, or maybe benchmark the hardware to pick better defaults, but this kind of thing is too difficult to get right for the benefit it provides. I'd love to try it someday, but definitely not for v1.4 which is already way overdue, so I'm taking it off the milestone.

**Comment by StarWitch** [comment]
Couldn't the "benchmark" be as simple as loading the background stages, gauging their performance, and then adjusting the render quality for background rendering until it gets a smooth 60 fps (with a bit of overhead for things like bombs/spellcards)? 

Even if it's not perfect, it'd still be a step up from someone starting the game and not knowing why they're getting crap performance even though their system should be able to handle it if they just turned down the background draw resolution.
